build/gulpfile.vscode.js

```diff
@@ -62,6 +62,8 @@ const vscodeResources = [
 	'out-build/bootstrap-amd.js',
 	'out-build/bootstrap-window.js',
 	'out-build/paths.js',
+	'out-build/telemetry-core.json',
+	'out-build/telemetry-extensions.json',
```

Comment:
@lramos15 I noticed this code isn't in master anymore but it wasn't removed during the PR to remove the command. What happened to it?